dask/array/tests/test_optimization.py

```diff
@@ -117,13 +116,6 @@ def test_hard_fuse_slice_cases():
     assert optimize_slices(dsk) == {'x': (getarray, 'x', (None, 5))}
 
 
-def test_dont_fuse_different_slices():
```

Comment:
Because `len(dsk) > 100` ends false with this PR. I'm not sure how to make the test relevant again (short of choosing a new comparison value at random). Perhaps @mrocklin would like to chime in.